modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,15 @@
+== Inline formatting.
+
+List bullets and list numbers can be spot- or inline-formatted.
+
+If an end-user selects the contents of a list item and applies inline formatting, such as a change of color or change of font-size, this spot formatting is also applied to the list item’s associated bullet or number.
+
+Only the bullets or list numbers associated with the selection are inline-formatted.
+
+NOTE: PowerPaste does not, currently, support inline-formatted bullets or list numbers. Inline-formatting, as applied to the bullets or list numbers, is lost when such material is copied and pasted.
```

Comment:
This `Powerpaste` looks like it's completely without context. Is there an`asciidoc attribute` or a xref that we should be using here. If I was using the community editor of TinyMCE, I think I'd be wondering what PowerPaste is.

Reply:
To, again, quote my commit message:

> Turned PowerPaste NOTE admonition into short section on PowerPaste (lack of) support.

Reply:
FYI: I’ve not _resolved this conversation_ because I’m still contemplating your question regarding turning this (now) section into a `/partial/`.

Reply:
At this point, I’m inclined to leave the PowerPaste section in place.

It pertains to PowerPaste’s support (or lack thereof) for inline formatting specifically, so I can’t see the section, as a whole, being re-used elsewhere.

And, while I can see the sub-head as well as the subject and verb of the first sentence being re-used, the prose becomes incredibly awkward if the object is hived off into a new line and paragraph of its own.